cmd/api-headers.go

```
@@ -95,10 +94,23 @@ func setObjectHeaders(w http.ResponseWriter, objInfo ObjectInfo, contentRange *h
 		w.Header().Set(k, v)
 	}
 
```

Let's not ignore the error we panic

---

Ok, this one fixed too.